content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -135,6 +135,7 @@ different Kubernetes components.
 | `ServiceNodeExclusion` | `false` | Alpha | 1.8 | |
 | `ServiceTopology` | `false` | Alpha | 1.17 | |
 | `StartupProbe` | `false` | Alpha | 1.16 | |
```

Comment:
pls fill the last cell on line 137 with `1.17` which is the last release in which `StartupProbe` was still alpha.

Reply:
Done